rslib/src/storage/card/mod.rs

```diff
@@ -770,15 +773,18 @@ impl fmt::Display for ReviewOrderSubclause {
             ReviewOrderSubclause::EaseDescending => "factor desc",
             ReviewOrderSubclause::DifficultyAscending => "extract_fsrs_variable(data, 'd') asc",
             ReviewOrderSubclause::DifficultyDescending => "extract_fsrs_variable(data, 'd') desc",
-            ReviewOrderSubclause::RelativeOverdueness { today } => {
-                temp_string = format!("ivl / cast({today}-due+0.001 as real)", today = today);
+            ReviewOrderSubclause::RetrievabilitySm2 { today, order } => {
+                temp_string = format!(
+                    "ivl / cast({today}-due+0.001 as real) {order}",
```

Comment:
This is fine. But, for a more accurate sorting, we can calculate the reciprocal and then sort in the opposite direction.

So, it would become something like cast({today}-due as real) / ivl

Reply:
No objections to a more accurate implementation, but I'm nervous to change working code so close to a release - perhaps this could be fixed in a follow-up.